syncer/metrics.go

```diff
@@ -215,19 +241,46 @@ var (
 			Name:      "heartbeat_update_error",
 			Help:      "number of error when update heartbeat timestamp",
 		}, []string{"server_id"})
+
+	finishedTransactionTotal = metricsproxy.NewCounterVec(
```

Comment:
what's the expected usage of this metric?

Reply:
YL said `Showing DM replication of incremental data data to downstream per second transactions from all latitudes`

Reply:
I see, we'll use it as a "rate"